tests/unit/amo/components/TestSearchSuggestion.js

```diff
@@ -75,7 +76,7 @@ describe(__filename, () => {
     };
     const root = shallowComponent(props);
 
-    expect(root.find(Icon)).toHaveLength(2);
+    expect(root.find(IconRecommendedBadge)).toHaveLength(1);
     expect(root.find('.SearchSuggestion-icon-recommended')).toHaveLength(1);
```

Comment:
We don't really need two assertions here anymore, as they are both asserting about the same element being present. There were two before because the first one asserted that there were *two* `Icon`s.

Reply:
ah, ok